src/functions-and-arrays.js

```diff
@@ -29,9 +83,34 @@ const wordsUnique = [
   'bring'
 ];
 
+const uniquifyArray = array =>{
```

Comment:
All tests are passed, great job! You might be interested in this [thread](https://stackoverflow.com/questions/9229645/remove-duplicate-values-from-js-array/41318862), showing similar and different solutions. :)